master/types.html

```diff
@@ -1623,7 +1623,7 @@ <h2 id="DOMInterfacesForReflectingSVGAttributes">DOM interfaces for reflecting S
 </ol>
 
 <p>When a <a>reflected</a> content attribute is to be
-<dfn id="TermReserialize">reserialized</dfn>, optionally using a
+<dfn id="TermReserialize" data-dfn-type="dfn" data-dfn-export>reserialized</dfn>, optionally using a
```

Comment:
Is that a term we want to get referenced from outside? Seems very generic. But of course, another spec could always clarify which spec it wants to link to in case the term exists already.